kubernetes.go

```diff
@@ -175,7 +175,9 @@ func (c *Client) GetKubernetesClusters(id string) (*KubernetesItem, error) {
 	}
 
 	kubernetes := &KubernetesItem{}
-	err = json.NewDecoder(bytes.NewReader(resp)).Decode(kubernetes)
+	if err := json.NewDecoder(bytes.NewReader(resp)).Decode(kubernetes); err != nil {
```

Comment:
Does this compile? `err` is already defined on https://github.com/civo/civogo/blob/4cda645e52606880b3e5364af9f6a5e03b32c1e9/kubernetes.go#L172